packages/app/src/components/DynamicRoot/ScalprumRoot.tsx

```diff
@@ -27,19 +28,23 @@ const ScalprumRoot = ({
 }) => {
   const { loading, value } = useAsync(
     async (): Promise<{
-      appConfig: AppConfig[];
+      dynamicPlugins: DynamicPluginConfig;
       baseUrl: string;
       scalprumConfig?: AppsConfig;
     }> => {
       const appConfig = overrideBaseUrlConfigs(await defaultConfigLoader());
-      const reader = ConfigReader.fromConfigs(appConfig);
+      const reader = ConfigReader.fromConfigs([
+        ...appConfig,
+        baseFrontendConfig ?? { context: '', data: {} },
```

Comment:
Does this contain also the definition of the default tabs ?
If this is a base configuration that can possibly be overridden, shouldn't we do:

```ts
      const reader = ConfigReader.fromConfigs([
        baseFrontendConfig ?? { context: '', data: {} },
        ...appConfig,
     ]);
```

Reply:
Or maybe it won't change anything ?

Reply:
Ah yes, you are correct, this is meant to be overridden and so should be first, let me update that.